canvas.gypi

```diff
@@ -0,0 +1,40 @@
+{
+    'variables':
+    {
+        'has_FBDev%': 'true',
+        'has_X11%': '<!(./util/has_lib.sh X11)'
```

Comment:
I tried to do it that way, but don't know why it failed, so I decided to do it easy, we could be able to review it later... Also, I was thinking it would be possible to unify both static and dynamic checkins independtly for each library, but it would makes it more difficult and would need some time to do it...